config/paths-common.conf

```diff
@@ -61,3 +61,6 @@ dovecot_log = %(syslog_mail_warn)s
 solidpop3d_log = %(syslog_local0)s
 
 mysql_log = %(syslog_daemon)s
+
+# Directory with ignorecommand scripts
+ignorecommands_dir = /etc/fail2ban/filter.d/ignorecommands
```

Comment:
what for if it is hardcoded in setup.py anyways... and I guess jail.conf wouldn't be appropriate level anyways (fail2ban.conf would be I Guess... but let's just hardcode for now?)